docs/content/docs/deployment/config.md

```diff
@@ -479,6 +479,26 @@ Flink does not use Pekko for data transport.
 
 {{< generated/akka_configuration >}}
 
+### Flink Format Options
```

Comment:
I'm afraid this is not the right place to list format options. This page shows all configurations that can be used in `conf/flink-conf.yaml`(now renamed to `conf/config.yaml`), which controls the behavior of Flink cluster. Formats are only used in some Table / SQL connectors such as file system and Kafka, and there are already pages for them. Take CSV as an example:

https://github.com/apache/flink/blob/558ca75da2fcec875d1e04a8d75a24fd0ad42ccc/docs/content/docs/connectors/table/formats/csv.md

Reply:
Thanks for the info. If so, I think this PR is not really needed. I will close it.